src/_utils/hard-source-webpack-plugin/index.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import { join } from "path"
+import findCacheDir from "find-cache-dir"
+import HardSourceWebpackPlugin from "hard-source-webpack-plugin"
+import ObjectHash from "node-object-hash"
+
+export const getCacheDir = (config: PhenomicConfig): string => {
+  // Calculate hash based on config object
+  const hasher = new ObjectHash()
+  const hash = hasher.hash(config)
+
+  const cacheDir = findCacheDir({ name: "phenomic-hard-source-wp/" + hash })
```

Comment:
we should put everything under "phenomic/*". So more `"phenomic/hard-source-webpack/" + hash `. (wp is confusing)